mlir/test/Integration/lit.local.cfg

```diff
@@ -57,4 +64,9 @@ aarch64_mcr_cmd = configure_aarch64_mcr_cmd()
 #                                             support cross-compilation
 # * ARM_EMULATOR_UTILS_LIB_DIR - AArch64 native utilites library to support
 #                                cross-compilation.
+
+# Check for above options and set correct executables and runtime libs
+aarch64_mcr_cmd = configure_aarch64_mcr_cmd()
```

Comment:
I would take this one step further and refactor `configure_aarch64_mcr_cmd` to also run:
```python
config.substitutions.append(("%mcr_aarch64_cmd", aarch64_mcr_cmd))
```

This way both methods would be more consistent and the overall code hygiene would improve :)